packages/atomic/src/components/commerce/atomic-commerce-search-box/e2e/page-object.ts

```diff
@@ -14,14 +14,44 @@ export class AtomicCommerceSearchBoxLocators {
     return this.page.getByRole('combobox', {name: 'Search'});
   }
 
+  get clearButton() {
+    return this.page.getByLabel('Clear', {exact: true});
+  }
+
+  get ariaLive() {
+    return this.page.locator('div[id*="aria-live-"][id$="-search-box"]');
```

Comment:
Could it be more simply: 
```suggestion
    return this.page.locator('aria-live');
```
?

Reply:
This would select the full atomic-aria-live element but what I need is the div specific to the searchbox